### PR TITLE
Jscourse2

### PR DESCRIPTION
**Pull request by Ajay-singh1**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
This PR is targeted to add Hindi/Urdu Courses in the repo which are free.

### Description
This PR is mainly helpful for all the people who wish to understand javascript programming language in Hindi?urdu.
### How do we know it's really free?
It is available on youtube free of cost.
### For book lists, is it a book? For course lists, is it a course? etc.
This is a course and not a book.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Ajay-singh1** [comment]
@eshellman Kindly Preview this PR and merge it.

File: courses/free-courses-ur.md [hunk 1]
```diff
@@ -12,3 +13,9 @@
 ### HTML and CSS
 
 * [HTML5 & CSS3 Tutorials In Urdu and Hindi](https://youtube.com/playlist?list=PLUyYwyJA_WfTr3YWWJ41_V7TrRZoq6cBT) - ProgramInUrdu
+
+
+### Javascript
+
+* [Javascript Tutorials In Urdu and Hindi](https://www.youtube.com/watch?v=Lgxgm-T9cgA&list=PL0b6OzIxLPbx-BZTaWu_AF7hsKo_Fvsnf) - Yahoo Baba
```
Comment (sohansai) [inline courses/free-courses-ur.md:20]: For YouTube, we prefer "playlist" links, not individual video links.

The link you provided seems different from the title.

Are you referring to this link: https://www.youtube.com/playlist?list=PL0b6OzIxLPbx-BZTaWu_AF7hsKo_Fvsnf

And use the original title from the resource and avoid inventing new ones.

File: courses/free-courses-ur.md [hunk 2]
```diff
@@ -12,3 +13,9 @@
...
+* [Learn JavaScript in Urdu & Hindi - For Beginners - Introduction](https://www.youtube.com/watch?v=NKgdmmiPVUc&list=PL52B8F99BC6AD1647) - WP Academy
```
Comment (sohansai) [inline courses/free-courses-ur.md:21]: Are you referring to this link: https://www.youtube.com/playlist?list=PL52B8F99BC6AD1647

If so, use the original title from the source and avoid creating new ones.

Comment (Ajay-singh1) [reply courses/free-courses-ur.md:21]: Ok Man!

File: courses/free-courses-ur.md (hunk 1 above)
Comment (Ajay-singh1) [reply courses/free-courses-ur.md:20]: Working on it.

**Comment by Ajay-singh1** [comment]
Working on #10407.

**Comment by Ajay-singh1** [comment]
Everything is done on #10419

**Pull request closed without merging**